### DISCUSSION TITLE
SIP-0003 discussion

### DISCUSSION THREAD

**Discussion by Semisol** [stackchainsats/stackchain#5] [General]
See: https://github.com/stackchainsats/stackchain/blob/main/sips/sip-0003.txt
Proposer: @Semisol 
Twitter thread: https://twitter.com/Semisol_Public/status/1549911068566224901
Supersedes: SIP-0002

**Comment by derekross** [comment]
Proof of concept going up here: https://twitter.com/derekmross/status/1549914858971439106

**Reply by Semisol** [reply]
Confirmed at height 229

**Comment by derekross** [comment]
If we don't have any objections objections, I'm okay with merging this SIP.

**Reply by DanjaminDW** [reply]
Merge. Successful proof of concept.
